packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -700,10 +700,15 @@ export class PhoenixObjects {
     });
     // object
     const sphere = new Mesh(geometry, material);
-    sphere.position.x = vertexParams.pos[0];
-    sphere.position.y = vertexParams.pos[1];
-    sphere.position.z = vertexParams.pos[2];
-
+    if ('pos' in vertexParams) {
+      sphere.position.x = vertexParams.pos[0];
+      sphere.position.y = vertexParams.pos[1];
+      sphere.position.z = vertexParams.pos[2];
+    } else {
+      sphere.position.x = vertexParams.x;
+      sphere.position.y = vertexParams.y;
+      sphere.position.z = vertexParams.y;
```

Comment:
Why z coordinate of a sphere equals y coordinate of the vertex?

Reply:
oups... because I reintroduced the code that was removed some time ago, and reintroduced the bug that was there altogether ! Fixing it right now